LICENSE

```diff
@@ -633,8 +633,8 @@ the "copyright" line and a pointer to where the full notice is found.
     Copyright (C) <year>  <name of author>
```

Comment:
Is this where I put the year and name then?

Reply:
Is there no place in the license file where I mention the year and name then?

Reply:
@nesdis re:

> Is there no place in the license file where I mention the year and name then?

That's correct, there is no place in the text which is explicitly designed to be left as-is and not modified.
You put your copyright notice with some license notice outside and elsewhere as instructed in https://github.com/nesdis/djoango/blob/master/LICENSE#L621
Typically in every file or you can use something simpler like the linux kernel approach (I have helped there a bit) 

See https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/license-rules.rst and https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/Documentation/process/license-rules.rst?id=aa19a176df95d6e49295d6ff77f7967224c71761

See also the excellent guides from https://reuse.software/